robot_manager:
  ros__parameters:
    control_mode: 1
    position_controller_name: "joint_trajectory_controller"
    impedance_controller_name: "joint_group_impedance_controller"
    torque_controller_name: "effort_controller"
# Control mode enums:
#  1 - joint position control
#  2 - joint impedance control
#  3 - joint velocity control (not supported yet)
#  4 - joint torque control
# ... others are not yet supported
